dbms/src/TestUtils/FunctionTestUtils.h

```diff
@@ -807,7 +811,8 @@ class FunctionTest : public ::testing::Test
         const ColumnNumbers & argument_column_numbers,
         const ColumnsWithTypeAndName & columns,
         const TiDB::TiDBCollatorPtr & collator = nullptr,
-        bool raw_function_test = false);
+        bool raw_function_test = false,
```

Comment:
Are these changes still needed?

Reply:
> Are these changes still needed?

Yes, we still need them to pass meta data in ut.